.storybook/Select.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Select, Label, Box } from '../src'
+
+storiesOf('Select', module)
+  .add('default', () => (
+    <Box>
+      <Label htmlFor="cabinClass">Cabin Class</Label>
+      <Select id="cabinClass" name="cabinClass" defaultValue="Premium Economy">
```

Comment:
For my education, where is the support code for the `defaultValue` attribute? I read up on the MDN `<select>` [doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) thinking that it's a native html attribute, but it doesn't seem to be referenced. Though my local testing via story confirms that `defaultValue` attribute works as expected. +1

Reply:
The `defaultValue` is part of React's form API – it allows the select to change in the storybook without the need for state management. We probably won't be using `defaultValue` in our apps or other components, since we use controlled components with the `value` and `onChange` props.

https://reactjs.org/docs/uncontrolled-components.html